src/utl/include/utl/Logger.h

```diff
@@ -327,3 +327,40 @@ class Logger
 #undef GENERATE_STRING
 
 }  // namespace utl
+
+// Define stream_as for fmt > v10
+#if !SWIG && FMT_VERSION >= 10000
+
+namespace utl {
+
+namespace {
+
+struct test_ostream {
+public:
+  template <class T>
+  static auto test(int) -> decltype(std::declval<std::ostream>() << std::declval<T>(), std::true_type());
+
+  template <class>
+  static auto test(...) -> std::false_type;
+};
+
+} // anonymous namespace
+
+template <class T, class = std::enable_if_t<decltype(test_ostream::test<T>(0))::value>>
+auto format_as(T t) { return fmt::streamed(t); }
+
+} // namespace rtl
+
+#include <boost/asio/ip/address.hpp>
+namespace boost::asio::ip {
+inline static auto format_as(const boost::asio::ip::address f) { return fmt::streamed(f); }
```

Comment:
Why does this need to be here as most of the program doesn't need to depend on asio?

Reply:
It can also be moved to the cpp file which uses asio. There are two file that depends on this:
- dst/src/LoadBalancer.cc
- dst/src/LoadBalancer.cc

I will move this piece of code into those two files.

Reply:
Updated in 928fe1a375ab0630fe97490ba4b1e9bd06ef7aa2